react/file_watcher.py

```diff
@@ -27,10 +32,18 @@ def __init__(self, base_path, watch_patterns=None, ignore_patterns=None):
         self.last_modified_file = None
         self.last_modified_file_inc = 0
 
+    def display_content(self, dir_path):
+        print("Watching:")
+        folders_and_files = sorted(pathlib.Path(dir_path).iterdir(), key=lambda p: (not p.is_dir(), p.name))
+        if(len(folders_and_files) == 0):
```

Comment:
If notation without `(...)`
```suggestion
        if len(folders_and_files) == 0:
```